classes/autoloader.php

```diff
@@ -317,7 +317,7 @@ protected function getClassAlias($class)
 
     protected function handleNamespaceOfClass($class)
     {
-        foreach ($this->directories as $namespace => $directories) {
+        foreach (array_keys($this->directories) as $namespace) {
```

Comment:
This is not efficient. `array_keys` allocates a new array. Keep `$namespace => $_` for the binding, it will be OK.